src/components/Article_Card.jsx

```diff
@@ -0,0 +1,41 @@
+import '../styles/ArticleCard.css'
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+const Article = () => {
+    const { article_id } = useParams();
+    const [article, setArticle] = useState(null);
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        const fetchArticle = async () => {
+            try {
+                const response = await fetch(`http://localhost:9090/api/articles/${article_id}`);
+                if (!response.ok) throw new Error('Failed to fetch article');
+                const data = await response.json();
+                setArticle(data.article);
+                setLoading(false);
+            } catch (error) {
+                console.error(error);
+                setLoading(false);
+            }
```

Comment:
It would be better to abstract the api call function into another file. Also, please take a look at axios, it's neater and the error message will be more helpful than fetch.